src/clobber/module_dependencies.txt

```diff
@@ -0,0 +1,10 @@
+analyses
+ansi-c
+cbmc
+clobber
+cpp
+goto-programs
+goto-instrument
+java_bytecode # will go away
```

Comment:
What's the difference between "will go away" and "should go away"?

Reply:
'will go away' has already a PR filed ;)